### ISSUE TITLE
[Keyboard Navigation - Logic Apps - Workflow settings]: "IP ranges for contents" text field is not accessible through keyboard.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6105]
### Describe the issue

User Experience:
Users who rely on keyboard for navigation will be impacted if text field is not accessible through keyboard as they will not be able to access the text field and will not be able to navigate properly.

Note: User credentials should NOT be included in the bug.

Repro Steps:
- Open https://ms.portal.azure.com/ and login via valid credentials.
- Navigate to the Left Nav bar and select 'More Services', search 'Logic Apps' and select it. Select any already created logic app.
- Select 'Work Flow Settings' in the left navigation pane.
- Navigate all the controls on the 'Work Flow Settings' blade.
- Verify whether "IP ranges for contents" text field is accessible through keyboard.

Actual Result:
The "IP ranges for contents" text field is not accessible through keyboard. Keyboard focus moves from "Trigger access option" combo box to the "Select an integration account" info button directly.

Expected Result:
The "IP ranges for contents" text field should be accessible through keyboard also.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Chrome, Edge

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29399968

**Comment by hartra344** [comment]
This control is appropriately keyboard accessible
